### PR TITLE
Added more headers and cleaned up commands section

### PR DESCRIPTION
**Pull request by alanngo**
check terminal/linux_macos-terminal/README.md

### CODE REVIEW

File: terminal/linux_macos-terminal/README.md [hunk 1]
```diff
@@ -7,19 +7,68 @@ The challenge with teaching anyone how to use the terminal is the rather large l
...
+    - Example: `ls` -> `README.md`, since this README is the only folder in the linux_macos-terminal folder. 
+- **touch**: Creates a file
+    - Example: `touch thisisatest.txt` -> creates a file named thisisatest.txt in the current directory
+- **mkdir**: Makes a directory (folder)
+    - Example: `mkdir this_is_a_folder` -> creates a folder named this_is_a_folder in the current directory. 
+- **rm**: "remove". This command is used to remove files and folders from the terminal. NOTE: Use this command with caution, since deleted files cannot be recovered easily. 
+    - Example 1: `rm thisisatest.txt` -> this removes the text file you created with `touch` in the last example
+    - Example 2: `rm -r this_is_a_folder` -> the `-r` flag means "recursive", and is necessary to delete folders. This command deletes the folder you created in a previous example. 
+
+# Using Makefiles
```
Comment (kdesai2018) [inline terminal/linux_macos-terminal/README.md:24]: lets save this section to include somewhere else - people don't use makefiles just when interacting with the terminal. we could potentially create a section focusing on different programming languages and you could include this in a c/cpp guide

DELETE makefile section

File: terminal/linux_macos-terminal/README.md [hunk 2]
```diff
@@ -7,19 +7,68 @@ The challenge with teaching anyone how to use the terminal is the rather large l
...
+
+terminal: (w/o arguments)
+	$ make tag: executes commands under tag
+
+terminal: (w/ arguments)
+	$ make tag1 ARG=<VALUE>: executes commands under tag1 w/ specified arguments
+
+# Installing Packages
+_______________________________________________________
+apt install <PACKAGE>: installs <PACKAGE>
```
Comment (kdesai2018) [inline terminal/linux_macos-terminal/README.md:41]:
```suggestion
## Linux package manager
apt install <PACKAGE>: installs <PACKAGE>
```

File: terminal/linux_macos-terminal/README.md [hunk 3]
```diff
@@ -7,19 +7,68 @@ The challenge with teaching anyone how to use the terminal is the rather large l
...
+	tag1:
+		command $(ARG)
+
+terminal: (w/o arguments)
+	$ make tag: executes commands under tag
+
+terminal: (w/ arguments)
+	$ make tag1 ARG=<VALUE>: executes commands under tag1 w/ specified arguments
+
+# Installing Packages
```
Comment (kdesai2018) [inline terminal/linux_macos-terminal/README.md:39]: could you also include a quick sentence or so describing what exactly a package is, and link them to the instructions on how to install brew for MacOS

File: terminal/linux_macos-terminal/README.md [hunk 4]
```diff
@@ -7,19 +7,68 @@ The challenge with teaching anyone how to use the terminal is the rather large l
...
+- "~": user home directory
+- ".": current working directory
+- "..": superdirectory of current working directory
+- "*": all files w/i working directory
+- "*.ext": all files w/i working directory w/ extension .ext
+- ">": writes the command to an output file
+- "<": reads input file into command
+- "sudo": run command as root user
+
+# Integrate this reference sheet w/ terminal
```
Comment (kdesai2018) [inline terminal/linux_macos-terminal/README.md:59]: where is commands.txt

File: terminal/linux_macos-terminal/README.md [hunk 5]
```diff
@@ -7,19 +7,68 @@ The challenge with teaching anyone how to use the terminal is the rather large l
...
+_______________________________________________________
+1. Open linux terminal
+2. $cd ~
+3. $touch commands.txt
+4. vim commands.txt
+5. paste this text into commands.txt using vim commands
+6. save and quit
+7. $vim .bashrc
+8. paste this command (w/o double quotes):  "alias commands = 'cat ~/commands.txt'"
```
Comment (kdesai2018) [inline terminal/linux_macos-terminal/README.md:68]:
```suggestion
8. paste this command (without the double quotes):  "alias commands = 'cat ~/commands.txt'"
```

File: terminal/linux_macos-terminal/README.md [hunk 6]
```diff
@@ -7,19 +7,68 @@ The challenge with teaching anyone how to use the terminal is the rather large l
...
+- "<": reads input file into command
+- "sudo": run command as root user
+
+# Integrate this reference sheet w/ terminal
+_______________________________________________________
+1. Open linux terminal
+2. $cd ~
+3. $touch commands.txt
+4. vim commands.txt
+5. paste this text into commands.txt using vim commands
```
Comment (kdesai2018) [inline terminal/linux_macos-terminal/README.md:65]: to talk about vim, you'd need to explain that first + give the reader instructions on how to install and tell them what exactly vim is

File: terminal/linux_macos-terminal/README.md (hunk 4 above)
Comment (alanngo) [reply terminal/linux_macos-terminal/README.md:59]: commands.txt is what the users create in their home directory via touch command, It is then used in "alias commands = 'cat ~/commands.txt" where users can create aliases to pull up the ref sheet in the terminal

**Comment by alanngo** [comment]
made new revisions

File: terminal/linux_macos-terminal/README.md (hunk 2 above)
Comment (alanngo) [reply terminal/linux_macos-terminal/README.md]: Resolved

File: terminal/linux_macos-terminal/README.md (hunk 4 above)
Comment (alanngo) [reply terminal/linux_macos-terminal/README.md]: Im going to address this issue in office hrs. For now, I will leave it off

**Pull request merged**